guide/the-user-interface.md

```diff
@@ -46,16 +46,23 @@ filename entered by the user. Note that a ".wbt" file should always be saved in
 a Webots project directory, and in the "worlds" subdirectory, otherwise it will
 not be possible to reopen the file.
 
-- ![](images/revert-button.png =26x26) The **Revert World** menu item (and button)
+- ![](images/revert-button.png =26x26) The **Reload World** menu item (and button)
 reloads the current world from the saved version and restarts the simulation
 from the beginning.
 
     > **Note**:
-When opening (or reverting) a world, if the world takes more than 2 seconds to
+When opening (or reloading) a world, if the world takes more than 2 seconds to
 load a window will pop up and show the progression. Using this window it is
 possible to cancel the loading of the world, in that case, the empty world will
 be opened instead.
 
+- ![](images/reset-simulation-button.png =26x26) The **Reset Simulation** menu item (and button) restore the initial state of the simulation.
+The simulation is not entirely destructed and reconstructed like with a reload, but the initial state of all the nodes is restored, which is way faster.
+
+    > **Note**:
+There are currently some limitations to the reset.
+The node removed from the simulation (either manually or from a Supervisor) are not re-imported during the reset. Furthermore, not all the changes to the node fields are reverted.
```

Comment:
Which are the node fields changes that are not reverted?
The user needs to know this information.